everest

```diff
@@ -65,6 +65,9 @@ cd_to_everest () {
   echo " ... now in $(pwd)"
   echo
 }
+# Save the initial working directory, to switch to when $0 pull calls
+# the new version of $0
+initial_pwd=$(pwd)
```

Comment:
if you intend to support paths with spaces, then double-quotes are probably needed here